Minio/Helper/s3utils.cs

```diff
@@ -35,22 +37,23 @@ internal static bool IsAmazonEndPoint(string endpoint)
         // the China (Beijing) Region. Customers with existing AWS credentials
         // will not be able to access resources in the new Region, and vice versa.
         // For more info https://aws.amazon.com/about-aws/whats-new/2013/12/18/announcing-the-aws-china-beijing-region/
-        internal static bool IsAmazonChinaEndPoint(string endpoint)
+        public static bool IsAmazonChinaEndPoint(string endpoint)
+
         {
 
             return endpoint == "s3.cn-north-1.amazonaws.com.cn";
```

Comment:
As `cn-north-1` region is hardcoded, when new region introduced in china can not work.  Isn't it possible to handle china as `s3.amazonaws.com.cn` ?